frontend/src/Components/Link/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useContext } from "react";
+import { usePlaidLink } from "react-plaid-link";
+import Button from "plaid-threads/Button";
+
+import Context from "../../Context";
+
+interface Props {
+  currentPath: string;
+}
+
+const Link: React.FC<Props> = (props: Props) => {
+  const {
+    itemId,
+    accessToken,
+    linkToken,
+    linkSuccess,
+    isItemAccess,
+    dispatch,
+  } = useContext(Context);
+
+  const onSuccess = React.useCallback(
+    (public_token: string) => {
+      // send public_token to server
+      const setToken = async () => {
+        const response = await fetch("/api/set_access_token", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8",
+          },
+          body: `public_token=${public_token}`,
+        });
+        if (!response.ok) {
+          dispatch({
+            type: "SET_STATE",
+            state: { itemId: `no item_id retrieved` },
+          });
+          dispatch({
+            type: "SET_STATE",
+            state: { accessToken: `no access_token retrieved` },
+          });
+          dispatch({ type: "SET_STATE", state: { isItemAccess: false } });
+          return;
+        }
+        const data = await response.json();
+        dispatch({ type: "SET_STATE", state: { itemId: data.item_id } });
+        dispatch({
+          type: "SET_STATE",
+          state: { accessToken: data.access_token },
+        });
+        dispatch({ type: "SET_STATE", state: { isItemAccess: true } });
+      };
+      setToken();
+      dispatch({ type: "SET_STATE", state: { linkSuccess: true } });
+      window.history.pushState("", "", "/");
+    },
+    [props]
+  );
+
+  let isOauth = false;
+  const config: Parameters<typeof usePlaidLink>[0] = {
+    token: linkToken!,
+    onSuccess,
+    clientName: "hello world",
```

Comment:
did we hardcode this in the old quickstart? If so, should we use the same value? Same question for products, should we be using the products defined in the environment variables? Did the old quickstart do that or was it hardcoded to auth/transactions there too?

Reply:
removed and left only token and onSuccess